Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [OuterTune/OuterTune strings](https://hosted.weblate.org/projects/outertune/app/).



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/outertune/app/horizontal-auto.svg)

lucasmz-dev commented:
@mikooomich I noticed you locked Weblate, are you having any issues, do you need help?

mikooomich commented:
??? I though I unlocked it. Well, long story short, did a force push, tried to reset it after (since i merged right before FP and there wasn't any new changes). I might have hit something during that causing whatever mess happen here, idk why Weblate is insisting these 13 commits should be here. @lucasmz-dev 


Edit: It was appearing locked on the translation page but unlocked in repo maintenance. I'll leave it locked for now until we can clear the extra commits

weblate committed:
Translated using Weblate (Hebrew)

Currently translated at 93.7% (238 of 254 strings)

Translated using Weblate (Polish)

Currently translated at 98.0% (249 of 254 strings)

Translated using Weblate (German)

Currently translated at 93.7% (238 of 254 strings)

Translated using Weblate (Vietnamese)

Currently translated at 39.7% (101 of 254 strings)

Translated using Weblate (Spanish)

Currently translated at 98.0% (249 of 254 strings)

Translated using Weblate (Greek)

Currently translated at 35.4% (90 of 254 strings)

Translated using Weblate (Bulgarian)

Currently translated at 98.0% (249 of 254 strings)

Translated using Weblate (Azerbaijani)

Currently translated at 26.7% (68 of 254 strings)

Translated using Weblate (Chinese (Simplified Han script))

Currently translated at 92.9% (236 of 254 strings)

Translated using Weblate (Russian)

Currently translated at 98.0% (249 of 254 strings)

Translated using Weblate (Japanese)

Currently translated at 98.0% (249 of 254 strings)

Translated using Weblate (Ukrainian)

Currently translated at 98.0% (249 of 254 strings)

Translated using Weblate (French)

Currently translated at 98.0% (249 of 254 strings)

Translated using Weblate (Italian)

Currently translated at 98.0% (249 of 254 strings)

Translated using Weblate (Portuguese (Brazil))

Currently translated at 98.0% (249 of 254 strings)

Translated using Weblate (English)

Currently translated at 100.0% (254 of 254 strings)

Co-authored-by: Anonymous <noreply@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/outertune/app/az/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/bg/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/de/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/el/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/en/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/es/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/fr/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/he/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/it/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/ja/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/pl/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/pt_BR/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/ru/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/uk/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/vi/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/zh_Hans/
Translation: OuterTune/OuterTune strings

lucasmz-dev committed:
Translated using Weblate (Vietnamese)

Currently translated at 39.7% (101 of 254 strings)

Translated using Weblate (Greek)

Currently translated at 35.4% (90 of 254 strings)

Translated using Weblate (Azerbaijani)

Currently translated at 26.7% (68 of 254 strings)

Translated using Weblate (Chinese (Simplified Han script))

Currently translated at 92.9% (236 of 254 strings)

Translated using Weblate (English)

Currently translated at 100.0% (2 of 2 strings)

Translated using Weblate (English)

Currently translated at 100.0% (254 of 254 strings)

Co-authored-by: LucasMZ <git@lucasmz.dev>
Translate-URL: https://hosted.weblate.org/projects/outertune/app-fastlane/en/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/az/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/el/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/en/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/vi/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/zh_Hans/
Translation: OuterTune/App store metadata
Translation: OuterTune/OuterTune strings

mikooomich committed:
Translated using Weblate (Vietnamese)

Currently translated at 39.7% (101 of 254 strings)

Translated using Weblate (Greek)

Currently translated at 35.4% (90 of 254 strings)

Translated using Weblate (Azerbaijani)

Currently translated at 26.7% (68 of 254 strings)

Translated using Weblate (English)

Currently translated at 100.0% (254 of 254 strings)

Co-authored-by: Michael Zh <mikooomichael@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/outertune/app/az/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/el/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/en/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/vi/
Translation: OuterTune/OuterTune strings

Mickael81 committed:
Translated using Weblate (French)

Currently translated at 98.0% (249 of 254 strings)

Co-authored-by: Mickaël Binos <mickaelbinos@outlook.com>
Translate-URL: https://hosted.weblate.org/projects/outertune/app/fr/
Translation: OuterTune/OuterTune strings

mryx007 committed:
Translated using Weblate (German)

Currently translated at 93.7% (238 of 254 strings)

Co-authored-by: mryxd774405165544858 <mryx@mail.de>
Translate-URL: https://hosted.weblate.org/projects/outertune/app/de/
Translation: OuterTune/OuterTune strings

SONGARKSOS committed:
Translated using Weblate (Korean)

Currently translated at 90.9% (231 of 254 strings)

Co-authored-by: 송의찬 <arksos1357@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/outertune/app/ko/
Translation: OuterTune/OuterTune strings

weblate committed:
Translated using Weblate (Sindhi)

Currently translated at 0.7% (2 of 254 strings)

Translated using Weblate (Thai)

Currently translated at 14.1% (36 of 254 strings)

Translated using Weblate (Swedish)

Currently translated at 2.7% (7 of 254 strings)

Translated using Weblate (Portuguese)

Currently translated at 98.0% (249 of 254 strings)

Translated using Weblate (Catalan)

Currently translated at 17.7% (45 of 254 strings)

Translated using Weblate (Indonesian)

Currently translated at 93.7% (238 of 254 strings)

Translated using Weblate (Serbian)

Currently translated at 90.1% (229 of 254 strings)

Translated using Weblate (Chinese (Traditional Han script))

Currently translated at 44.8% (114 of 254 strings)

Translated using Weblate (Persian)

Currently translated at 93.7% (238 of 254 strings)

Translated using Weblate (Hindi)

Currently translated at 69.6% (177 of 254 strings)

Translated using Weblate (Estonian)

Currently translated at 46.8% (119 of 254 strings)

Translated using Weblate (Arabic)

Currently translated at 80.7% (205 of 254 strings)

Translated using Weblate (English (Canada))

Currently translated at 1.5% (4 of 254 strings)

Translated using Weblate (Korean)

Currently translated at 90.9% (231 of 254 strings)

Co-authored-by: Anonymous <noreply@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/outertune/app/ar/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/ca/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/en_CA/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/et/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/fa/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/hi/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/id/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/ko/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/pt/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/sd/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/sr/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/sv/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/th/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/zh_Hant/
Translation: OuterTune/OuterTune strings

lucasmz-dev committed:
Translated using Weblate (Estonian)

Currently translated at 46.8% (119 of 254 strings)

Translated using Weblate (Arabic)

Currently translated at 80.7% (205 of 254 strings)

Translated using Weblate (English (Canada))

Currently translated at 1.5% (4 of 254 strings)

Translated using Weblate (Korean)

Currently translated at 90.9% (231 of 254 strings)

Co-authored-by: LucasMZ <git@lucasmz.dev>
Translate-URL: https://hosted.weblate.org/projects/outertune/app/ar/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/en_CA/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/et/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/ko/
Translation: OuterTune/OuterTune strings

mikooomich committed:
Translated using Weblate (Chinese (Traditional Han script))

Currently translated at 44.8% (114 of 254 strings)

Translated using Weblate (Hindi)

Currently translated at 69.6% (177 of 254 strings)

Translated using Weblate (Arabic)

Currently translated at 80.7% (205 of 254 strings)

Translated using Weblate (English (Canada))

Currently translated at 1.5% (4 of 254 strings)

Co-authored-by: Michael Zh <mikooomichael@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/outertune/app/ar/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/en_CA/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/hi/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/zh_Hant/
Translation: OuterTune/OuterTune strings

lucasmz-dev committed:
Translated using Weblate (Thai)

Currently translated at 14.1% (36 of 254 strings)

Translated using Weblate (Swedish)

Currently translated at 2.7% (7 of 254 strings)

Translated using Weblate (Catalan)

Currently translated at 17.7% (45 of 254 strings)

Translated using Weblate (Serbian)

Currently translated at 90.1% (229 of 254 strings)

Translated using Weblate (Chinese (Traditional Han script))

Currently translated at 44.8% (114 of 254 strings)

Co-authored-by: LucasMZ <git@lucasmz.dev>
Translate-URL: https://hosted.weblate.org/projects/outertune/app/ca/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/sr/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/sv/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/th/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/zh_Hant/
Translation: OuterTune/OuterTune strings

mikooomich committed:
Translated using Weblate (Swedish)

Currently translated at 2.7% (7 of 254 strings)

Translated using Weblate (Catalan)

Currently translated at 17.7% (45 of 254 strings)

Translated using Weblate (Serbian)

Currently translated at 90.1% (229 of 254 strings)

Co-authored-by: Michael Zh <mikooomichael@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/outertune/app/ca/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/sr/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/sv/
Translation: OuterTune/OuterTune strings

weblate committed:
Update translation files

Updated by "Remove blank strings" add-on in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/outertune/app/
Translation: OuterTune/OuterTune strings

lucasmz-dev commented:
i think we fell victim to a bug in this new feature, i used that new option unknowningly, https://github.com/WeblateOrg/weblate/issues/16751

i'll need to check later what we might have lost when it comes to diffs and outdated strings
